goal_topic: /nav_robot/move_base_simple/goal
is_reached_goal_topic: waypoint/is_reached
regist_goal_pose_topic: waypoint/regist_pose
regist_goal_point_topic: waypoint/regist_point
visualize_waypoints_topic: waypoints
#robot_base_frame: robot1_tf/base_footprint
robot_base_frame: nav_robot/base_footprint
global_frame: nav_robot/map
goal_publish_frequency: 0.1 #0.5
goal_check_frequency: 1.0 #1.0
default_goal_radius: 1.0

waypoints_file: $(find waypoint_server)/config/waypoints/racing.yaml
route_file: $(find waypoint_server)/config/routes/racing.yaml


enable_loop: true
debug: false
